js/app.js

```diff
@@ -1 +1,98 @@
-console.log('Hello frontend');
+document.addEventListener('DOMContentLoaded', () => {
+    const squares = document.querySelectorAll('.grid-container div')
+    const playerDisplay = document.querySelector('#player')
+    const cellOne = document.querySelector('#one')
+    const cellTwo = document.querySelector('#two')
+    const cellThree = document.querySelector('#three')
+    const cellFour = document.querySelector('#four')
+    const cellFive = document.querySelector('#five')
+    const cellSix = document.querySelector('#six')
+    const cellSeven = document.querySelector('#seven')
+    const cellEight = document.querySelector('#eight')
+    const cellNine = document.querySelector('#nine')
+    
+    
+    let currentPlayer = 'Player X'
+    squares.forEach(square => {
+        square.addEventListener('click', clickOutcome)
+});
+
+
+    function clickOutcome(e) {
+        const squareArray = Array.from(squares)
+        const index = squareArray.indexOf(e.target)
+        playerDisplay.innerHTML = currentPlayer
+        console.log(index);
+
+        if(currentPlayer === 'Player X'){
+            squares[index].classList.add('playerX')
+            currentPlayer = 'Player O'
+        } else {
+            currentPlayer = 'Player X'
+            squares[index].classList.add('playerO')
+        }
```

Comment:
Take a look at my general comments for the pull request regarding the first major point in regards to this section.  See how you are adding the class and changing the currentPlayer without noting whether a square has been played on yet or not?